client/src/components/custom/ErrorToast.tsx

```diff
@@ -17,17 +19,17 @@ const ErrorToast: React.FC = () => {
   useEffect(() => {
     if (!errorMessage) return;
     if (previousAct) clearTimeout(previousAct);
-    const closeMethod = setTimeout(activeMessage, 1500);
+    const closeMethod = setTimeout(activeMessage, TOAST_TIME);
     setPreviousAct(closeMethod);
     setDisplayMessage(errorMessage);
     dispatch(setNoticeMessage({ errorMessage: '' }));
```

Comment:
resetNoticeMessage와 같이 초기값으로 초기화 하는 액션을 별도로 만들어 쓰는 것이 좋아보입니다!
```suggestion
    dispatch(resetNoticeMessage());
```

Reply:
헉 너무 좋은 것 같습니다!! 다음주에 리팩토링 하면서 적용해보도록 하겠습니다!!

Reply:
항상 '' 형태로 초기화하는 것만 사용했었는데, 생각의 지평을 열어주셨군요..! 감사합니다 :)